core/src/main/java/org/apache/gravitino/audit/AuditLogConfig.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.audit;
+
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.gravitino.Config;
+import org.apache.gravitino.config.ConfigBuilder;
+import org.apache.gravitino.config.ConfigConstants;
+import org.apache.gravitino.config.ConfigEntry;
+import org.apache.gravitino.utils.MapUtils;
+
+public class AuditLogConfig extends Config {
+  public static final String AUDIT_LOG_ENABLED = "enable";
+  static final String AUDIT_LOG_WRITER_PREFIX = "writer.";
+  static final String AUDIT_FORMATTER_PREFIX = "formatter.";
+  static final String AUDIT_LOG_WRITER_CLASS_NAME = AUDIT_LOG_WRITER_PREFIX + "class";
+  static final String AUDIT_LOG_FORMATTER_CLASS_NAME = AUDIT_FORMATTER_PREFIX + "class";
+
+  static final ConfigEntry<Boolean> AUDIT_LOG_ENABLED_CONF =
+      new ConfigBuilder(AUDIT_LOG_ENABLED)
+          .doc("Gravitino audit log enable flag")
```

Comment:
Whether to enable Gravitino audit log ?

Reply:
What is the configuration name here? I just saw "enabled", is it correct? If it is a configuration, it should be like "gravitino.audit.enabled", something like this.

Reply:
I would suggest to move all the configuration definitions to class `Configs`, it is not necessary to have a separate place to maintain the configurations.

Also, please use the full name to manage the configurations for simplicity, and you can define the `AuditLogManager#initialize(Config config, xxx)` like this. You don't help to remove the prefix and make things complicated.